genesis.md

```diff
@@ -481,3 +481,40 @@ same principle can be used to scale up other aspects of
 deployments, such as CPU, memory, or disk sizes — just keep in
 mind this may need to be done indirectly by specifying instance or
 disk types and adding the desired resources in your cloud-config.
+
+## Stemcell Version Upgrades
+
+Genesis provides a simple way for you to update stemcells for deployments based off of Genesis Kits.  The typical process is to upload a new stemcell to the BOSH Director and perform a `genesis deploy` on each deployment.
+
+In each kit there is a default for the os flavor and version:
+
+```
+params:
+  stemcell_os:      ubuntu-trusty
+  stemcell_version: latest
+```
+
+To upgrade the stemcell version for a deployment start by uploading a new stemcell to the BOSH Director.  Full instuctions for doing this are located in the [BOSH Runbook under Uploading a Stemcell](bosh.md#upload-a-stemcell).
+
```

Comment:
You will not be able to link to the `*.md` files;  Check https://runbooks.starkandwayne.com for the URI to use to skip to another runbook.

Reply:
Updated to use full url